plugins/PrescaleToCommon_miniAOD.cc

```diff
@@ -100,12 +100,12 @@ bool PrescaleToCommon_miniAOD::filter(edm::Event& event, const edm::EventSetup&
   unsigned path_index = hlt_cfg.size();
   
  
-    const std::vector<std::string>& pathList = hlt_cfg.triggerNames();
-    std::cout<<"path size "<<pathList.size()<<std::endl;
+    //const std::vector<std::string>& pathList = hlt_cfg.triggerNames();
+    //std::cout<<"path size "<<pathList.size()<<std::endl;
```

Comment:
Can we have these as flags for extra debugging information, or is the information provided by the list not relevant anymore?

Reply:
Judging from the commit history of this file (the non-MiniAOD version), the stl output from this file has been in flux and is likely used mainly for development/debugging purposes. A flag for the stl output with the default value set to False is probably better than just commenting the lines out (it makes it clear for future users what the output's purpose is).

Reply:
Yes I would agree to have it go to a debugging stream with the debugging set to False by default (this is what I meant earlier).